src/Microsoft.ML.ImageAnalytics/ImageResizerTransform.cs

```diff
@@ -44,7 +44,10 @@ public enum ResizingKind : byte
             IsoPad = 0,
 
             [TGUI(Label = "Isotropic with Cropping")]
-            IsoCrop = 1
+            IsoCrop = 1,
+
```

Comment:
Would be great if you could a summary tag above these options, this will make it show in the documentation online and in the intellisense for future users.

```suggestion

/// <summary>
/// Ignore aspect ratio and squeeze/stretch into target dimensions.
/// </summary>
```